plugins/cue_points/base/lib/model/CuePointPeer.php

```diff
@@ -365,4 +365,17 @@ public static function validateMetadataObjectAccess($objectId)
 	
 		return true;
 	}
+
+	public static function setCuePointsStatus($ids, $newStatus)
```

Comment:
Should be changed to deleteCuePoints($ids, $entryId)
The entryID is for protection.
I see no case where we make vast change of cuepoints status plus it can open case for issues.